src/DssSpell.sol

```diff
@@ -65,312 +70,293 @@ contract DssSpellAction is DssAction {
+        // Forum: https://forum.sky.money/t/utilization-of-the-integration-boost-budget-a-5-2-1-2/25536/2
+        // Poll: https://vote.makerdao.com/polling/QmYHUDVA
+
+        // Transfer 3,000,000 USDS to 0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7
+        _transferUsds(INTEGRATION_BOOST_INITIATIVE, 3 * MILLION * WAD);
+
+        // ---------- ESM Minimum Threshold increase ----------
+        // Forum: https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579
+
+        // Increase ESM Minimum Threshold by 200,000 MKR, from 300,000 MKR to 500,000 MKR
+        DssExecLib.setValue(MCD_ESM, "min", 500 * THOUSAND * WAD);
+
+        // ---------- October 2024 AD Compensation ----------
+        // Forum: https://forum.sky.money/t/october-2024-aligned-delegate-compensation/25581
         // Atlas: https://sky-atlas.powerhouse.io/A.1.5.8_Budget_For_Prime_Delegate_Slots/e3e420fc-9b1f-4fdc-9983-fcebc45dd3aa%7C0db3af4ece0c
 
-        // BLUE - 13.75 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 13.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // BLUE - 2,968 USDS - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
+        _transferUsds(BLUE,        2_968 * WAD);
+
+        // Bonapublica - 4,000 USDS  - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
+        _transferUsds(BONAPUBLICA, 4_000 * WAD);
+
+        // Byteron - 1,733 USDS  - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
+        _transferUsds(BYTERON,     1_733 * WAD);
 
-        // Cloaky - 29.25 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 29.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Cloaky - 4,000 USDS  - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
+        _transferUsds(CLOAKY,      4_000 * WAD);
 
-        // JuliaChang - 28.75 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 28.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // JuliaChang - 4,000 USDS  - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
+        _transferUsds(JULIACHANG,  4_000 * WAD);
 
-        // Byteron - 9.68 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 9.68 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // vigilant - 4,000 USDS  - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
+        _transferUsds(VIGILANT,    4_000 * WAD);
 
-        // vigilant - 2.43 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
-        MKR.transfer(VIGILANT, 2.43 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // ---------- Atlas Core Development Payments ----------
+        // Forum: https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580
+        // Atlas: https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/8
```

Comment:
Please either use the `Forum:` prefix here or remove this line as it links to the same post as the line above :)

Reply:
Done.